### PR TITLE
Add com.jpexs.decompiler.flash

### PR DESCRIPTION
**Pull request by Newbytee**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** I have emailed upstream and received their blessing to handle this.
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

~~I have one issue however: It doesn't find the icon! I'm not sure why, thus this is a draft for now. Suggestions welcome.~~ Fixed, my path was wrong.

Also, please add @jindrapetrik as co-maintainer of the repository once this is merged.

### CODE REVIEW

**Comment by Newbytee** [comment]
bot, build

**Comment by Newbytee** [comment]
bot, build com.jpexs.decompiler.flash

**Comment by Newbytee** [comment]
bot, build com.jpexs.decompiler.flash

File: com.jpexs.decompiler.flash.json
Comment (hfiguiere) [inline com.jpexs.decompiler.flash.json:14]: why `host`?

File: com.jpexs.decompiler.flash.json
Comment (hfiguiere) [inline com.jpexs.decompiler.flash.json:49]: it's GPL software, why not rebuilding from source and add aarch64 support? Does this package even contain non Java binaries since there seem to be a java only zip.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.jpexs.decompiler.flash.json
Comment (Newbytee) [reply com.jpexs.decompiler.flash.json:14]: This is largely a personal preference. I sometimes want to access files outside of my home directory, and this software doesn't support portals. That said, I understand if home is preferable here, and I'm open to changing it.

File: com.jpexs.decompiler.flash.json
Comment (Newbytee) [reply com.jpexs.decompiler.flash.json:49]: Oh, you're thinking it should be usable on aarch64 too since there's no native code? That is very possible.

**Comment by barthalion** [comment]
bot, build com.jpexs.decompiler.flash

File: com.jpexs.decompiler.flash.json
Comment (barthalion) [reply com.jpexs.decompiler.flash.json:49]: Let's do it. At first glance it looks like it should work just fine, although I haven't checked on ARM hardware.

Comment (Newbytee) [reply com.jpexs.decompiler.flash.json:49]: > Let's do it

Could you clarify what you're proposing I should do?

Comment (barthalion) [reply com.jpexs.decompiler.flash.json:49]: I think removing `flathub.json` will be enough.

**Comment by Newbytee** [comment]
bot, build com.jpexs.decompiler.flash

**Comment by Newbytee** [comment]
I'm going to test the aarch64 build on my PinePhone. It's installing right now, I'll post here when I have results.

**Comment by Newbytee** [comment]
photo_2021-08-17_15-04-55

Works!

**Comment by Newbytee** [comment]
I think this is ready for merge now. I understand it may be desirable to build from source. I'm not opposed to this at all, but I'm not sure how to handle Ant, and I can't find any docs about how to use it in Flatpak. I may investigate this later.

**Comment by Newbytee** [comment]
And, as a reminder, please add @jindrapetrik as co-maintainer of the repository after merge.

**Pull request closed without merging**
